src/structures/command/parameters/types/MessageLinkParameter.js

```diff
@@ -43,18 +46,24 @@ module.exports = class MessageLinkParameter extends Parameter {
         const channel = guild.channels.cache.get(channelId)
         if (!channel) throw new CommandError(t('errors:validLinkButGhostChannel'))
 
-        try {
-          const recievedMessage = await channel.messages.fetch(messageId)
-          if (!recievedMessage) throw new CommandError(t('errors:validLinkButGhostMessage'))
+        const recievedMessage = await channel.messages.fetch(messageId).catch()
```

Comment:
This is not what I meant.. You still need to pass a function on the `catch` method.
```suggestion
        const recievedMessage = await channel.messages.fetch(messageId).catch(() => null)
```